### PR TITLE
#12748: Cleanup clamp_bw op

### LINKED ISSUE #12748: Cleanup 3 Unary Backward ops
Master : #12721

- [x] ExecuteUnaryBackwardNeg
- [x] ExecuteUnaryBackwardRsqrt
- [x] ExecuteUnaryBackwardOptionalFloatParamsWithDefault

### PR DESCRIPTION
**Pull request by VirdhatchaniKN**
Issue : #12748
Test File : `tests/ttnn/unit_tests/operations/backward/test_backward_clamp.py`
https://github.com/tenstorrent/tt-metal/actions/runs/10901615365 - Passed

### CODE REVIEW

File: ttnn/cpp/ttnn/operations/eltwise/unary_backward/device/unary_backward_op.cpp [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 namespace ttnn::operations::unary_backward {
 
-std::vector<Tensor> _clamp_bw(
+std::vector<Tensor> ExecuteUnaryBackwardClampBW::invoke(
```
Comment (KalaivaniMCW) [inline ttnn/cpp/ttnn/operations/eltwise/unary_backward/device/unary_backward_op.cpp:31]: ExecuteUnaryBackwardClamp ?

**Review by KalaivaniMCW — APPROVED** [review]
Approved. pls address the minor change mentioned

File: ttnn/cpp/ttnn/operations/eltwise/unary_backward/device/unary_backward_op.cpp (hunk 1 above)
Comment (VirdhatchaniKN) [reply ttnn/cpp/ttnn/operations/eltwise/unary_backward/device/unary_backward_op.cpp]: Addressed

**Review by eyonland — APPROVED** [review]

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/10914664085 - passed

**Pull request merged**